src/test/mocks/archiveLoader.ts

```diff
@@ -84,7 +84,7 @@ export default () => {
     mockPage: (parentBook: ArchiveBook, newPage: ArchivePage, pageSlug: string) => {
       localBookPages[`${parentBook.id}@${parentBook.version}`][newPage.id] = newPage;
       localBooks[`${parentBook.id}@${parentBook.version}`].tree.contents.push({
-        id: `${newPage.id}@${newPage.version}`,
+        id: `${newPage.id}@${newPage.revised}`,
```

Comment:
it might be more realistic to just hardcode this to like `@1.0` or something, at least then it'll be the right format